gnucash/register/register-gnome/gnucash-sheet.c

```diff
@@ -789,6 +789,29 @@ gnucash_sheet_finalize (GObject *object)
     g_object_unref (sheet->im_context);
 }
 
+#define GLYPH_PAPERCLIP "\360\237\223\216"
+#define GLYPH_LINK      "\360\237\224\227"
+
+gboolean
+gnucash_sheet_use_glyphs (GnucashSheet *sheet)
```

Comment:
Can't this code be made a function of assoccell instead of sheet so that the determination of whether to use them is colocated with the code that does use them. Speaking of which, I don't see any code that actually enables using them.

Reply:
I did not think it could but having tried, yes it can so moved.